### DISCUSSION TITLE
Edit how bard menu items behave

### DISCUSSION THREAD

**Discussion by nicktc** [statamic/cms#4808] [Q&A]
Is it possible that the Bard toolbar will not insert `<h1>..</h1>` by clicking the `<h1>` button, but will insert something like `<h1` class="text-3xl font-semibold"></h1>? I am using Tailwind, so all tags are not styled (or all styling is cleared by Tailwind). So a `<h1>` will look exactly the same as a `<p>`.

Another question, is it not possible to type edit the HTML in a Bard field?

**Comment by jasonvarga** [comment]
It's not possible unless you introduce custom bard nodes which could get complicated.

I suggest you use the tailwind typography plugin which is designed to style cms-generated content.

https://tailwindcss.com/docs/plugins#typography

```html
<div class="prose">
  {{ bard_field }}
</div>
```

> Another question, is it not possible to type edit the HTML in a Bard field?

You cannot edit the html. Bard is designed so it generates consistent HTML for you.
